### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by jonathan-sha**

### CODE REVIEW

**Review by shaydeci — DISMISSED** [review]
lgtm

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment:**
+ - OS [e.g. uname -s -r -m]
+ - Relevant HW info, GPU [e.g. nvidia-smi]
```
Comment (jonathan-sha) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: Add cuda version

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ - Super Gradients version
+
```
Comment (RanZilberstein) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: I would like to see information about the python libraries installed - like running pip [freeze](https://pip.pypa.io/en/stable/cli/pip_freeze/) command. This way we can reproduce the bug in exact environment.

**Review by RanZilberstein — CHANGES_REQUESTED** [review]
Added a small comment to fix + Please look at the .md view format, you might want to add new lines and change the appearance a bit.

**Review by RanZilberstein — APPROVED** [review]
lgtm

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (avideci) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I think nvidia-smi has cuda version

**Review by shaydeci — APPROVED** [review]

**Pull request merged**